.github/workflows/workflow.yml

```diff
@@ -7,91 +7,160 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2
+      - name: "1. Clone repository"
+        uses: actions/checkout@v2
 
-      - name: Set up Python 3.6
+      - name: "2. Set up Python 3.6"
         uses: actions/setup-python@v1
         with:
           python-version: 3.6
```

Comment:
We could even use 3.8 here

Reply:
I was thinking to keep as low as we can, so we can have bigger audience. Should we test library with multiple python versions?

Reply:
That would be nice :D

Reply:
can do a matrix of python versions and run the github action on each version :) i.e. it's supported by github actions

Reply:
unrelated to this PR but relevant, would also try decreasing pinned requirements which currently kinda force the last version of everything

Reply:
iirc with 3.8 there were some issues with libraries, so please make sure at least the example notebooks from Docker run :)

Reply:
looks like it's possible to test if notebooks execute without errors: https://www.blog.pythonlibrary.org/2018/10/16/testing-jupyter-notebooks/

could add to our testing action; probably different PR though :)

Reply:
@radujica I will do pinned requirements in different PR
@TheMP @hrzn will add matrix of 3.6, 3.7 and 3.8 pythons for current tests